circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala

```diff
@@ -16,11 +16,70 @@ import io.circe._
 import io.circe.jawn.JawnParser
 import org.reflections8.Reflections
 
+/**
+ * An abstract class, that is extended to create a custom serializer.
```

Comment:
Nit, [no comma before `that`](https://virtuslab.github.io/tips/#english/grammar-tips-2)